src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/WhitespaceAroundCheck.java

```diff
@@ -213,6 +213,35 @@
  * <pre>
  * &lt;module name=&quot;WhitespaceAround&quot;/&gt;
  * </pre>
+ * <p>Example:
+ * </p>
+ * <pre>
+ * class Test {
+ *     public Test(){} // 2 violations , '{' is not followed and preceded by whitespace.
+ *     public static void main(String[] args) {
+ *         if (foo) { // ok
+ *             // body
+ *         }
+ *         else{ // violation
+ *             // body
+ *         }
+ *
+ *         for (int i = 1; i &gt; 1; i++) {} // violation ,
+ *                                             // '{' is not followed by whitespace.
```

Comment:
please make `//` horiznotally aligned.
Please align all comments like this.

Reply:
Done with the changes @romani

Reply:
please update images in PR description and please make sure you aligned all such multiline comments

Reply:
Hey @romani  updated the website and the changes are done